lib/Organizations.php

```diff
@@ -93,19 +95,20 @@ public function createOrganization($name, $domains, $allowProfilesOutsideOrganiz
      * Update Organization.
      *
      * @param string $organization An Organization identifier.
-     * @param array $domains The domains of the Organization.
+     * @param array $domains [Deprecated] The domains of the Organization. Use domain_data instead.
+     * @param array $domain_data The domains of the Organization.
      * @param string $name The name of the Organization.
      * @param null|boolean $allowProfilesOutsideOrganization Whether Connections within the Organization allow profiles
      *      that are outside of the Organization's configured User Email Domains.
      *
      * @throws Exception\WorkOSException
      */
-    public function updateOrganization($organization, $domains, $name, $allowProfilesOutsideOrganization = null)
+    public function updateOrganization($organization, $domains = null, $name = null, $allowProfilesOutsideOrganization = null, $domain_data = null)
     {
         $organizationsPath = "organizations/{$organization}";
         $params = [
-          "organization" => $organization,
```

Comment:
The organization ID is embedded in the path and is not a param.

Reply:
Is this safe to remove? Will we break an existing user's integration? I'd prefer if this were a minor version release.

Reply:
Yep, this is safe. This parameter was never needed (for the reason Oliver mentioned) and won't introduce any breakage.